pkgs/top-level/all-packages.nix

```diff
@@ -29484,6 +29484,8 @@ with pkgs;
 
   dfasma = libsForQt5.callPackage ../applications/audio/dfasma { };
 
+  dfilemanager = libsForQt5.callPackage ../applications/file-managers/dfilemanager { };
```

Comment:
I don't know why this is removed while `pkgs/applications/file-managers/dfilemanager/default.nix` remains